### ISSUE TITLE
add-referenceset in repo manager needs supplied metadata fields

### ISSUE THREAD

**Issue by macieksmuga** [ga4gh/server#1150]
When I issue a request to add a reference set:
```
python repo_dev.py add-referenceset -n hg37 ga4gh-example-data/repo.db ../maciek-ga4gh-compliance/test-data/hg37.fa
```
The resulting entity is missing several important fields:
```
{"name": "hg37", "sourceURI": null, "assemblyId": null, "sourceAccessions": [], "ncbiTaxonId": null, "isDerived": 0, "id": "WyJoZzM3Il0", "md5checksum": "12827085103f42e97428dc05a7f26fec", "description": ""}
```
most importantly the `assemblyId`, which can be searched by.

**Comment by diekhans** [comment]
The schema does allow these fields to be omitted.  is there no way to set them
with a metadata json file?

**Comment by jeromekelleher** [comment]
We need to add CLI options for these metadata fields. It's a 10 minute job.

**Comment by dcolligan** [comment]
Fixed in #1184
